sound/soc/sof/sof-audio.c

```diff
@@ -625,7 +625,7 @@ int sof_set_up_pipelines(struct snd_sof_dev *sdev, bool verify)
 	int ret;
 
 	/* restore pipeline components */
-	list_for_each_entry_reverse(swidget, &sdev->widget_list, list) {
+	list_for_each_entry(swidget, &sdev->widget_list, list) {
```

Comment:
this was designed by @ranj063 .  I also asked why but I didn't remember reply

Reply:
let's ask @ranj063